### PR TITLE
OADP-4138: Attributes update for OADP 1.3.2 and Velero relationship update

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [OADP-4138](https://issues.redhat.com/browse/OADP-4138)

### Version

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16

### Link to docs preview:

* [OADP-Velero-OpenShift Container Platform version relationship](https://76482--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator#velero-oadp-version-relationship_installing-oadp-operator)

* [OADP support for IBM Power and IBM Z](https://76482--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins#oadp-support-for-ibm-power-and-ibm-z) - **As this should be updated for each release of OADP 1.1.x, OADP 1.2.x & OADP 1.3.x - can we do something like this?** 

### QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 1]
```diff
@@ -21,9 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
...
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
-* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
-* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
+* {oadp-short} {oadp-version-1-1} was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]:
```suggestion
* {oadp-short} {oadp-version-1-1} was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-1} in terms of backup locations for these systems.
```

**Review by weshayutin — APPROVED** [review]
/LGTM

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 2]
```diff
@@ -21,9 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
...
 
-* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
-* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
-* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
+* {oadp-short} {oadp-version-1-1} was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-1} in terms of backup locations for these systems.
```
Comment (weshayutin) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]: OADP-1.1 is no longer supported, no need to add new comments regarding it.

File: modules/oadp-ibm-power-test-support.adoc [hunk 3]
```diff
@@ -6,6 +6,6 @@
...
-* {ibm-power-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.12, 4.13. 4.14, and 4.15, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
-* {ibm-power-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
+* {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} {oadp-version-1-1} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} {oadp-version-1-1} against all S3 backup location targets, which are not AWS, as well.
```
Comment (weshayutin) [inline modules/oadp-ibm-power-test-support.adoc:9]: same here, no need to comment on OADP-1.1.x

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc (hunk 2 above)
Comment (anarnold97) [reply backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]: So comment out or leave with the new `oadp-version-1-1` attributes, please?

thanks

File: modules/oadp-ibm-z-test-support.adoc [hunk 4]
```diff
@@ -6,6 +6,6 @@
...
-* {ibm-z-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.12, 4.13, 4.14 and 4.15, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
-* {ibm-z-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 4.14, and 4.15, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
+* {ibm-z-name} running with {product-title} 4.11 and 4.12, and {oadp-first} {oadp-version-1-1} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and {oadp-short} {oadp-version-1-1} against all S3 backup location targets, which are not AWS, as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-z-test-support.adoc:9]:
```suggestion
// * {ibm-z-name} running with {product-title} 4.11 and 4.12, and {oadp-first} {oadp-version-1-1} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and {oadp-short} {oadp-version-1-1} against all S3 backup location targets, which are not AWS, as well.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc (hunk 2 above)
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]:
```suggestion
// * {oadp-short} {oadp-version-1-1} was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-1} in terms of backup locations for these systems.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 5]
```diff
@@ -21,9 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
...
 
-* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
-* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
-* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
+// * {oadp-short} {oadp-version-1-1} was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} {oadp-version-1-1} in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]:
```suggestion
```

File: modules/oadp-ibm-power-test-support.adoc [hunk 6]
```diff
@@ -6,6 +6,6 @@
...
-* {ibm-power-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.12, 4.13. 4.14, and 4.15, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
-* {ibm-power-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
+// * {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} {oadp-version-1-1} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} {oadp-version-1-1} against all S3 backup location targets, which are not AWS, as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-power-test-support.adoc:9]:
```suggestion
```

File: modules/oadp-ibm-z-test-support.adoc [hunk 7]
```diff
@@ -6,6 +6,6 @@
...
-* {ibm-z-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.12, 4.13, 4.14 and 4.15, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
-* {ibm-z-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 4.14, and 4.15, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
+// * {ibm-z-name} running with {product-title} 4.11 and 4.12, and {oadp-first} {oadp-version-1-1} was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and {oadp-short} {oadp-version-1-1} against all S3 backup location targets, which are not AWS, as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-z-test-support.adoc:9]:
```suggestion
```

File: _attributes/common-attributes.adoc [hunk 8]
```diff
@@ -38,7 +38,9 @@ endif::[]
 :oadp-first: OpenShift API for Data Protection (OADP)
 :oadp-full: OpenShift API for Data Protection
 :oadp-short: OADP
-:oadp-version: 1.3.1
+:oadp-version: 1.3.2
+:oadp-version-1-2: 1.2.4
```
Comment (stillalearner) [inline _attributes/common-attributes.adoc:42]: 1.2.5 now?

File: modules/velero-oadp-version-relationship.adoc [hunk 9]
```diff
@@ -17,5 +17,8 @@
 | 1.2.1 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.2.2 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.2.3 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
+| 1.2.4 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
```
Comment (stillalearner) [inline modules/velero-oadp-version-relationship.adoc:20]: 1.2.5 now?

Comment (anarnold97) [inline modules/velero-oadp-version-relationship.adoc:20]:
```suggestion
| 1.2.4 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
| 1.2.5 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
```

**Comment by anarnold97** [comment]
@stillalearner - please can you re-review? thanks

**Pull request closed without merging**